src/main/resources/application-local.yml

```diff
@@ -12,16 +12,23 @@ spring:
 soap:
   xsd-file-path: src/main/resources/xsd/cp/external/ExternalDocumentRequest.xsd
 
-# Localstack settings
+hmpps.sqs:
+  enabled: true
+  provider: localstack
+  queues:
+    courtcaseeventsqueue:
+      queueName: court_case_events_queue
+      subscribeTopicId: courtcaseeventstopic
+      dlqName: cpr_court_case_events_queue_dlq
+      dlqMaxReceiveCount: 1
```

Comment:
Queue is only there for verification purposes - the app as deployed does not require it